getting_started_tutorial_final_v3_api/run_checkpoint.py

```diff
@@ -0,0 +1,11 @@
+import great_expectations as ge
+
+# add great_expectations/plugins to path
+import sys, os
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'great_expectations'))
```

Comment:
Note that without this script, trying to use the cli fails and it isn't clear to me how to use the cli with custom expectations:

```
getting_started_tutorial_final_v3_api$ great_expectations --v3-api checkpoint run my_checkpoint_with_custom_expectation
Using v3 (Batch Request) API
'NoneType' object is not callable
```